tests/test_enterprise/test_signals.py

```diff
@@ -532,3 +533,87 @@ def test_receiver_does_not_call_task_if_ecu_not_exists(self, mock_task):
 
         create_enterprise_enrollment_receiver(sender, instance, **kwargs)
         mock_task.assert_not_called()
+
+
+@mark.django_db
+class TestEnterpriseCatalogSignals(unittest.TestCase):
+    """
+    Tests the EnterpriseCustomerCatalogAdmin
+    """
+
+    @mock.patch('enterprise.signals.EnterpriseCatalogApiClient')
+    def test_delete_catalog(self, api_client_mock):
+        enterprise_catalog = EnterpriseCustomerCatalogFactory()
+        enterprise_catalog_uuid = enterprise_catalog.uuid
+        api_client_mock.return_value = mock.MagicMock()
+        api_client_mock.return_value.get_enterprise_catalog = mock.MagicMock()
+        api_client_mock.return_value.get_enterprise_catalog.return_value = True
+        api_client_mock.return_value.delete_enterprise_catalog = mock.MagicMock()
```

Comment:
Mocked objects will return mocks by default, so I _think_ you can condense all of this into
```python
api_client_mock.return_value.get_enterprise_catalog.return_value = True
```